hack/release.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/release.sh
+
+function build_release() {
+  local version="$(cat hack/NEXT_VERSION)-$(date -u +%Y%m%d)"
+  local now="$(date -u '+%Y-%m-%d %H:%M:%S')"
+  local rev="$(git rev-parse --short HEAD)"
+  local pkg="github.com/knative/client/pkg/kn/commands"
+  local ld_flags="-X '${pkg}.BuildDate=${now}' -X ${pkg}.Version=${version} -X ${pkg}.GitRevision=${rev}"
+
+  export GO111MODULE=on
+  export CGO_ENABLED=0
+  echo "🚧 🐧 Building for Linux"
+  GOOS=darwin GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-darwin-amd64 ./cmd/...
```

Comment:
./kn-linux-amd64

Reply:
Funny, the original code was swapped and we never realized it. :) Fixed, thanks.